resolwe/flow/managers/utils.py

```diff
@@ -0,0 +1,13 @@
+"""Utilities for using global manager features."""
+
+from django.test import override_settings
+
+
+def disable_auto_calls():
+    """Decorator/context manager which stops automatic manager calls.
+
+    When entered, automatic
+    :meth:`~resolwe.flow.manager.base.BaseManager.communicate` calls
+    from the django transaction signal are not done.
+    """
+    return override_settings(FLOW_MANAGER_STOP_AUTO_CALLS=True)
```

Comment:
I would rename `FLOW_MANAGER_STOP_AUTO_CALLS` to `FLOW_MANAGER_DISABLE_AUTO_CALLS`.